torchvision/ops/boxes.py

```diff
@@ -14,6 +14,9 @@ def nms(boxes, scores, iou_threshold):
     IoU greater than iou_threshold with another (higher scoring)
     box.
 
+    When all the values of the scores are the same, the results are 
```

Comment:
I think I would phrase it slightly differently, because this is a subset of the cases where this inconsistency happens.
I would maybe say something like:
> If multiple boxes have the exact same score and satisfy the IoU criterion with respect to a reference box, the selected box is not guaranteed to be the same between CPU and GPU. This is similar to the behavior of argsort in PyTorch when repeated values are present.

Reply:
thanks for the review, 

yeah agree, should also mention the IoU, the case is for boxes that overlap (> iou threshold) and the scores are all the same, will update it soon